src/main/java/kg/jarkyn/server/Incoming/Requester.java

```diff
@@ -0,0 +1,9 @@
+package kg.jarkyn.server.Incoming;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public interface Requester {
```

Comment:
I'm not sure with the naming here either way. ClientSocket and ServerSocket confused me initially, but that's probably because I'm not familiar with the networking lingo. Christoph probably has more experience with this so I'd trust his opinion :)